src/compile/selection/interval.ts

```diff
@@ -156,33 +126,47 @@ export function projections(selCmpt: SelectionComponent) {
   return {x, xi, y, yi};
 }
 
-function channelSignal(model: UnitModel, selCmpt: SelectionComponent, channel: Channel): any {
-  const name  = channelSignalName(selCmpt, channel),
+/**
+ * Returns the visual and data signals for an interval selection.
+ */
+function channelSignals(model: UnitModel, selCmpt: SelectionComponent, channel: Channel): any {
```

Comment:
The `VgSignal` type isn't fully descriptive yet. So rather than deal with compiler errors, I figured `any` was good enough for now. `any` is used in place of `VgSignal` throughout this part of the codebase, I think, so I would prefer to handle that separately from this PR (and perhaps as a lower-priority item for release) if you're amenable.